addons/mail/static/src/new/discuss/autogrow_input.js

```diff
@@ -10,13 +10,20 @@ export class AutogrowInput extends Component {
         });
         this.inputFocus = useFocus("input");
         this.inputRef = useRef("input");
-        this.rootHover = useHover("root");
+        this.inputHover = useHover("input");
         onWillUpdateProps((nextProps) => {
             if (this.props.value !== nextProps.value) {
                 this.state.value = nextProps.value;
             }
         });
         onExternalClick("input", () => this.onValidate());
+        useEffect(
+            () => {
+                this.inputRef.el.style.width = "1px";
+                this.inputRef.el.style.width = this.inputRef.el.scrollWidth + 5 + "px";
+            },
+            () => [this.state.value]
```

Comment:
I suggest taking the placeholder into account. So the width works correctly when only having place holder

Reply:
Actually `placeholder` was unused, so I just removed it.
If we really need it, then better introduce when we need.

I also removed `props.onDiscard` that was unused. Same reasoning: introduce only when we need it.

Reply:
The reason why I have the placeholder props is when we have an empty thread description, it can be helpful to have an "Add thread description" like that. I will add this feature when introducing the thread description editing feature

Reply:
Good idea, you can re-introduce placeholder for that feature. Thanks!